sql/dql/basic-selects/sql-practice-select-1.md

```diff
@@ -0,0 +1,43 @@
+---
+author: claremarie
+
+levels:
+  - beginner
+
+tags:
+  - introduction
+
+type: exercise
+
+linkType: sqlfiddle
+
+link: http://sqlfiddle.com/#!9/352b0f1/1
+
+answer: 4, 300, 5, 33, 4, heal-ball
```

Comment:
These will need a regex (and those that are similar)

Reply:
ok, done for all the new exercises I created